playbooks/roles/vault/tasks/main.yml

```diff
@@ -61,7 +61,7 @@
 
 - name: Install Vault environment variables
   lineinfile:
-    path: "{{ ansible_env.HOME }}/.bashrc"
+    path: "$HOME/.bashrc"
```

Comment:
So it looks like both of these actually spawn the changes into /root/.bashrc, whereas I thought they'd go into /home/ubuntu/.bashrc. I actually didn't know it'd go there in the first place, but I think that it's probably alright to go to root by default... although in practice it makes no difference because we aren't controlling security on those users so strictly.

Reply:
Good catch. Since the goal here is to ensure the logged-in user has the vault variables set, it would be a regression for people who login as the ubuntu user, right? I entirely missed that the first time around.

Reply:
Yeah it would be but one can easily `sudo -s` -> `source /root/.bashrc`. But yeah, it'd be nicer if we could just get it on the ubuntu user, since that's where all the other stuff is.

Reply:
I'll fix it then